heart/heartbeat.py

```diff
@@ -63,31 +75,29 @@ def calc_ib_version(w,x,y,a,z):
 		p = json.load(fp)
 		# Server priorities
 		for k, v in p["server_priorities"].iteritems():
-			if not isinstance(k, (str, unicode)) or not isinstance(v, int):
+			if not type(k) in (str, unicode) or not type(v) == int:
```

Comment:
`isinstance` is clearer, and the tiny performance hit (due to `isinstance` checking inheritance and `type() in` not) doesn't matter as it's only done once, on startup. I'd revert this.